spec/operators/smooth-spec.ts

```diff
@@ -0,0 +1,20 @@
+import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
+
+declare const { asDiagram };
+declare const hot: typeof marbleTestingSignature.hot;
+declare const cold: typeof marbleTestingSignature.cold;
+declare const expectObservable: typeof marbleTestingSignature.expectObservable;
+declare const expectSubscriptions: typeof marbleTestingSignature.expectSubscriptions;
+
+/** @test {smooth} */
```

Comment:
hmm the [other tests have them](https://github.com/ReactiveX/rxjs/blob/master/spec/operators/do-spec.ts#L14). Not sure what it's for. figured it was some sort of build time thing.